accounting/conversions.go

```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/lightninglabs/faraday/utils"
 )
 
-// usdPrice is a function which gets the USD price of bitcoin at a given time.
-type usdPrice func(timestamp time.Time) (*fiat.USDPrice, error)
+// fiatPrice is a function which gets the fiat price of bitcoin at a given time.
+type fiatPrice func(timestamp time.Time) (*fiat.Price, error)
```

Comment:
nit: some of these rename refactors could be split out into their own commits, makes things easy to catch.

They certainly should have been generically named in the first place, but here we are, sigh.